### PR TITLE
GF-2219 Updated examples to use new version of API (version 3)

### PR DESCRIPTION
**Pull request by claesahren**
Wait with merge until v.3 of API is live in production.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ For other installation methods, see: <https://github.com/swagger-api/swagger-cod
 ### 5. Generate the API Client
 
 ```shell
-swagger-codegen generate -i https://api.globalforester.com/swagger/v2/swagger.json -o Swagger -l python
+swagger-codegen generate -i https://api.globalforester.com/swagger/v3/swagger.json -o Swagger -l python
```
Comment (larsabrasha) [inline README.md:57]: Update text about versions at the top of this file?

**Review by larsabrasha — CHANGES_REQUESTED** [review]
Found a small thing in the README.md

File: README.md (hunk 1 above)
Comment (claesahren) [reply README.md:59]: Updated and added som information about previous api versions

**Review by larsabrasha — APPROVED** [review]
⭐️

**Pull request merged**